### PR TITLE
Add GEM/xsum prompts

### PR DESCRIPTION
**Pull request by kkawamu1**
Add 10 prompts for GEM/xsum.
All the prompts are taken from the prompts already implemented for https://huggingface.co/datasets/xsum in the host version promptsource.

### CODE REVIEW

**Comment by jzf2101** [comment]
@awebson from an engineering perspective do we have to copy these directly or could we pull from  https://huggingface.co/datasets/xsum as is?

**Comment by kkawamu1** [comment]
> @awebson from an engineering perspective do we have to copy these directly or could we pull from https://huggingface.co/datasets/xsum as is?

There is actually a slight difference in that https://huggingface.co/datasets/xsum  uses 'summary' as a key for the summary text,  while GEM/xsum uses 'target' as a key. So the prompts written for GEM/xsum reflect that difference. 

e.g. 

`{{document}}
      This boils down to the simple idea that ||| {{target}}`

instead of 

`{{document}}
      This boils down to the simple idea that ||| {{summary}}`

File: promptsource/templates.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 # These are users whose datasets should be included in the results returned by
 # filter_english_datasets (regardless of their metadata)
-INCLUDED_USERS = {"Zaid", "craffel"}
+INCLUDED_USERS = {"Zaid", "craffel", "GEM"}
```
Comment (awebson) [inline promptsource/templates.py:29]:
```suggestion
INCLUDED_USERS = {"Zaid", "craffel"}
```

Comment (awebson) [reply promptsource/templates.py:29]: do not commit this hack

**Review by awebson — APPROVED** [review]
Yup what Ken did is correct. We need to copy the old xsum prompts to gem/xsum and make necessary changes. Merging now since the xsum prompts were already approved. Thanks!

File: promptsource/templates.py (hunk 1 above)
Comment (awebson) [inline promptsource/templates.py]:
```suggestion
INCLUDED_USERS = {"Zaid", "craffel", "GEM"}
```

**Comment by awebson** [comment]
@kkawamu1 Looks like taking out the hack broke automatic tests :( And I somehow I don't have permission to commit a fix from my end. Can you re-add `GEM` to `INCLUDED_USERS = {"Zaid", "craffel", "GEM"}` and push again?

**Comment by kkawamu1** [comment]
@awebson  Should be updated! i.e. GEM is added to INCLUDED_USERS now!

**Pull request merged**